### DISCUSSION TITLE
MD run

### DISCUSSION THREAD

**Discussion by chaoyangma24** [YAMACS-SML/YAMACS#3] [General]
Thank for your kind reply.
I can run smoothly steps 1 and 2 for protein-ligand simulation according to demonstration in 4_protein_ligand_complex_acpype.webm,
but there appeared the following wrong when I run the third step (MD run). In fact, in my working directory (home/chaoyang/yasara), there was this file  md_o_1.xtc. I do not know why? 
![image](https://user-images.githubusercontent.com/83275350/229477654-46fb434e-aa0c-400d-b7e1-16617ece60fc.png)

**Comment by YAMACS-SML** [comment]
The problem may be the 'tempo.ini' file in the YASARA/plg/ folder. If there is any '**tempo.ini**' file (Note: Do not delete '**temporary.ini'** file), delete that file and re-run the 3rd plugin(MD-run). To make sure that the 'md_0_1.xtc' file is there you can increase the time in ine number 73 of YAMACS_3_MD.py file. By default it is 'time.sleep(2)'.  You can modify it with 'time.sleep(10)' and observe the error. Ii would be appreciated if you can kindly provide the YASARA terminal (Press space bar two times on YASARA window) images as well as the ubuntu terminal image while facing that error. From this image it is very tough to interpret the error. It seems YAMACS is looking for md_0_1.xtc file before it is generated.
